src/ducks/searchValue/reducers.js

```diff
@@ -0,0 +1,15 @@
+import {handleActions} from 'redux-actions'
+import {changeValue} from "./actions";
+
+export const initialState = {
+    value: '',
+}
+
+const searchValueReducer = {
+    [changeValue] : (state = initialState, action) => ({
+        ...state,
+        value: action.payload
+    })
+}
```

Comment:
use initialState = '' here instead of calliing properties of initialState as an object